.circleci/config.yml

```diff
@@ -13,13 +13,13 @@ install_js: &install_js
     name: Install js dependencies
     command: |
       yarn config set yarn-offline-mirror ~/.cache/npm-packages-offline-cache/v1
-      yarn
+      yarn --frozen-lockfile
```

Comment:
This option was not used on purpose. Instead, we display the git diff. What do you think?

Reply:
I had no changes to a yarn lock locally, but was failing on CI. Tried over again and same thing. So I'm not sure what the intent is here but I was unable to get it past the diff step regardless of what I did locally. 

Given this situation, we can continue with the diff method and potential false failures or just use something like renovate or greenkeeper and stick to a frozen lockfile. I would prefer to automate dependencies given our thorough test suite.

Reply:
I'm supposed you are referring to this failing step:

![capture d ecran 2019-01-01 a 13 11 43](https://user-images.githubusercontent.com/3165635/50572727-cddcdc00-0dc6-11e9-8842-9a4fe8df66fc.png)

I remember having this problem too at some point. If I recall correctly, it was an issue with my yarn version. The drawback of using frozen lockfile is that its making the issue mismatch hard to debug. At least, in my experience working with the two strategies.

Reply:
@eps1lon I know you have strong opinions on the CI. What do you think is the best strategy here?

Reply:
Because it does say what's wrong.

Reply:
FYI - my yarn version is up-to-date `1.12.3`.  Given I could not submit a PR that would pass - I'm not sure this diff step is something we can count on.

What is the argument against renovate or greenkeeper?  We would have a locked down package.json and PRs for every dependency update that get tested and auto-merged if successful.

Reply:
> my yarn version is up-to-date 1.12.3

I'm using v1.10.1. The CI is using 1.12.3. It might be something else than the yarn version then.

> What is the argument against renovate or greenkeeper?

It creates a lot of noise for very limited value in exchange. I find it way better to batch the updates, manually. It the type of project I would label false good idea. The selling pitch is great but the implementation horrible. Maybe it was just me not configuring it correctly 🤔.